src/frontend/utils/parseDataElements/parseDataElements.spec.ts

```diff
@@ -35,9 +38,19 @@ describe.only('utils/parseDataElements', () => {
       };
       dataElement.setAttribute('data-context', JSON.stringify(context));
 
-      expect(() => {
-        parseDataElements(dataElement);
-      }).toThrowError(`Model ${modelName.THUMBNAILS} not supported`);
+      let data: AppData;
+      let thrownError;
+      try {
+        data = parseDataElements(dataElement);
+      } catch (error) {
+        thrownError = error;
+      }
+
+      expect(thrownError).not.toEqual(
+        Error(`Model ${modelName.THUMBNAILS} not supported`),
+      );
```

Comment:
```suggestion
```

This `expect` is useless. You are testing right before that `thrownError` is undefined.